src/AutoUI/spec_tmp.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
Renamed to temporarly disable specs as jest is not happy with ui-shared-components build. Speaking about it we will probably remove jest and add playwright